repose-aggregator/components/filters/add-header-filter/src/test/scala/org/openrepose/filters/addheader/config/AddHeaderSchemaTest.scala

```diff
@@ -19,21 +19,20 @@
  */
 package org.openrepose.filters.addheader.config
 
+import java.net.URL
+
 import org.junit.runner.RunWith
-import org.openrepose.commons.test.ConfigValidator
+import org.openrepose.commons.test.ConfigurationTest
 import org.scalatest.junit.JUnitRunner
-import org.scalatest.{FunSpec, Matchers}
 import org.xml.sax.SAXParseException
 
 @RunWith(classOf[JUnitRunner])
-class AddHeaderSchemaTest extends FunSpec with Matchers {
-  val validator = ConfigValidator("/META-INF/schema/config/add-header.xsd")
+class AddHeaderSchemaTest extends ConfigurationTest {
+  override val schema: URL = getClass.getResource("/META-INF/schema/config/add-header.xsd")
+  override val exampleConfig: URL = getClass.getResource("/META-INF/schema/examples/add-header.cfg.xml")
+  override val jaxbContextPath: String = classOf[ObjectFactory].getPackage.getName
```

Comment:
Lol, you used that pointless ObjectFactory

Reply:
I did, kind of. I really just needed the bound package name, but hey, if I can get that from a consistently generated class, I'll take that over a string that would also be a second source of truth. :)